deps/chakrashim/src/v8handlescope.cc

```diff
@@ -54,28 +54,38 @@ HandleScope *HandleScope::GetCurrent() {
 }
 
 bool HandleScope::AddLocal(JsValueRef value) {
-  if (_count < _countof(_locals)) {
+  if (_count <= kOnStackLocals) {
```

Comment:
Once i reset _count to 0 at right place below, it should not AV anymore. I will address it once I fix failure scenario.